dubbo-plugin/dubbo-qos/pom.xml

```diff
@@ -89,7 +100,89 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.24.3</version>
```

Comment:
Delete all your added dependencies because these artifacts have already been added for testing.